tools/nebula-spark/src/main/java/com/vesoft/nebula/bean/Parameters.java

```diff
@@ -0,0 +1,11 @@
+package com.vesoft.nebula.bean;
+
+public class Parameters {
+    public static final String SPACE_NAME = "spaceName";
```

Comment:
Great projects have a short name. Such as MySQL Linux and Hadoop

Reply:
Great! The dataSource now has a shortName "nebula",  you can read nebula data by SparkSession.read().format("nebula").option......